lib/rufo/new_formatter.rb

```diff
@@ -266,13 +266,26 @@ def visit_begin(node)
 
   def visit_string_literal(node)
     # [:string_literal, [:string_content, exps]]
-    consume_token :on_tstring_beg
-    
-    inner = node[1..-1]
-    
+    if current_token_kind == :on_backtick
+      consume_token :on_backtick
+    else
+      consume_token :on_tstring_beg
+    end
+
+    visit_string_literal_end(node)
+  end
+
+  def visit_string_literal_end(node)
+    inner = node[1]
```

Comment:
Would you mind adding a comment to the beginning of this method showing what shape we expect? Copying from `visit_string_literal` should be sufficient